examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -1429,7 +1436,11 @@ impl FrontendNode {
 
             // Block until there is a response from one of the backends
             // available.
-            let readies = oak::wait_on_channels(&self.backend_in).map_err(OakError::OakStatus)?;
+            let readies = match oak::wait_on_channels(&self.backend_in) {
+                Ok(s) => Ok(s),
+                Err(ChannelStatusError::HasInvalid(s)) => Ok(s),
+                Err(ChannelStatusError::Error(s)) => Err(OakError::OakStatus(s)),
+            }?;
```

Comment:
Thanks. `or_else` reads better. 

The whole reason for returning the statuses as part of `HasInvalid` is to allow the caller code to access and use those statuses in its normal flow. So, `HasInvalid` should always be treated as `Ok`. This is why [I suggested](https://github.com/project-oak/oak/pull/907#discussion_r418323046) using `Result<ChannelStatusVec, OakStatus>` as the return type instead of returning `HasInvalid` as error, as it is currently done.